pkg/apis/kubeone/types.go

```diff
@@ -148,6 +148,11 @@ const (
 	// * In case when encryption is enabled, strong secret will be autogenerated
 	// More info: https://www.weave.works/docs/net/latest/kubernetes/kube-addon/
 	CNIProviderWeaveNet CNIProvider = "weave-net"
+
+	// CNIProviderExternal is an external CNI plugin .
```

Comment:
```suggestion
	// CNIProviderExternal is an external CNI plugin.
```
Tiny nit: space is not needed here

Reply:
fixed